gdnative-derive/src/methods.rs

```diff
@@ -218,7 +253,12 @@ fn impl_gdnative_expose(ast: ItemImpl) -> (ItemImpl, ClassMethodExport) {
                                     }
                                 };
 
-                                for MetaNameValue { path, .. } in pairs {
+                                for MetaNameValue {
+                                    path,
+                                    eq_token: _,
+                                    lit,
+                                } in pairs
```

Comment:
It's better to use `..` to non-exhaustively ignore extra fields here, in case new fields get added upstream.

Reply:
understood